File: VB/Controllers/HomeController.vb
Imports Microsoft.VisualBasic
Imports DevExpress.Web.Mvc
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Web
Imports System.Web.Mvc
Imports T283418.Models

Namespace T283418
	Public Class HomeController
		Inherits Controller
		Public Function Index() As ActionResult
			Return View()
		End Function
		Public Function GridViewPartial() As ActionResult
			Dim model = Data.GetData()
			Return PartialView(model)
		End Function
		Public Function GridViewBatchUpdatePartial(ByVal values As MVCxGridViewBatchUpdateValues(Of Integer, Data)) As ActionResult
			Dim model = Data.GetData()
			Return PartialView("GridViewPartial", model)
		End Function
	End Class
End Namespace

File: CS/Controllers/HomeController.cs
using DevExpress.Web.Mvc;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.Web.Mvc;
using T283418.Models;

namespace T283418
{
    public class HomeController : Controller
    {
        public ActionResult Index()
        {
            return View();
        }
        public ActionResult GridViewPartial()
        {
            var model = Data.GetData();
            return PartialView(model);
        }
        public ActionResult GridViewBatchUpdatePartial(MVCxGridViewBatchUpdateValues<int, Data> values)
        {
            var model = Data.GetData();
            return PartialView("GridViewPartial", model);
        }
    }
}